tiled/catalog/adapter.py

```diff
@@ -997,6 +1001,21 @@ def contains(query, tree):
     return tree.new_variation(conditions=tree.conditions + [condition])
 
 
+def full_text(query, tree):
+    dialect_name = tree.engine.url.get_dialect().name
+    conditions = []
```

Comment:
This pattern from other query implementations doesn't fit as well here, as we always add exactly one condition, as opposed to accumulating a list of conditions in loop.

Can probably just do

```py
condition = tsvector(...)
...
return tree.new_variation(conditions=tree.conditions + [condition])
```